client/main.lua

```diff
@@ -96,87 +93,56 @@ RegisterNUICallback('exit', function(_, cb)
     cb('ok')
 end)
 
-local LastRainbowNeonColor = 0
+local lastRainbowNeonColor = 0
 
-local RainbowNeonColors = {
-    [1] = {
-        r = 255,
-        g = 0,
-        b = 0
-    },
-    [2] = {
-        r = 255,
-        g = 165,
-        b = 0
-    },
-    [3] = {
-        r = 255,
-        g = 255,
-        b = 0
-    },
-    [4] = {
-        r = 0,
-        g = 0,
-        b = 255
-    },
-    [5] = {
-        r = 75,
-        g = 0,
-        b = 130
-    },
-    [6] = {
-        r = 238,
-        g = 130,
-        b = 238
-    },
+local rainbowNeonColors = {
```

Comment:
i didnt really look into it. i was mainly just focusing on adding the modules and doing minor style fixes. most of these scripts need to be gone through and updated anyway. didnt wanna worry about adding/shanging more stuff to a pr since you guys like small prs